types/mediawiki/MwLog.ts

```diff
@@ -0,0 +1,7 @@
+interface MwLog {
+	deprecate( obj: object, key: string, val: any, msg?: string, logName?: string ): void;
+	error( ...msg: any[] ): void;
+	warn( ...msg: string[] ): void;
+}
```

Comment:
[I think these types could be refined](https://gerrit.wikimedia.org/r/#/c/mediawiki/core/+/585781/). I'm happy to do that in a new PR if they're wanted. (These definitions are fine with me too :+1:)

Reply:
:+1: not an issue for me either way!